src/brain/issueLabelHandler/route.ts

```diff
@@ -235,5 +240,16 @@ export async function markRouted({
     body: comment,
   });
 
+  /**
+   * We'll try adding the issue to our global issues project. If it already exists, the existing ID will be returned
+   * https://docs.github.com/en/issues/planning-and-tracking-with-projects/automating-your-project/using-the-api-to-manage-projects#adding-an-item-to-a-project
+   */
+  const issueNodeId: string = await addIssueToProject(payload.issue.node_id, payload.repository.name, payload.issue.number, octokit);
```

Comment:
Isn't it the `itemId` that is returned here?

Reply:
good point, changed to `itemId`